src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -1,34 +1,43 @@
+</Callout>
 
-For organizations on the New Relic One account/user model, here are some details about how organization and account management work: 
+At New Relic, an "organization" is the representation of a New Relic customer. The organization contains everything pertinent to a New Relic customer: its accounts, its users, and its data. For organizations on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/), usage that impacts billing is aggregated at the organization level. 
 
-* The [Standard edition](https://newrelic.com/pricing) allows for a single account per organization. [Pro and Enterprise editions](https://newrelic.com/pricing) allows for [multiple accounts](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#users).
-* At the top level of a New Relic account hierarchy is the concept of the "organization." Your group of related accounts that share the same billing are inside a single "New Relic organization." 
-* Currently you can't add accounts to your organization on your own. To add accounts, talk to your New Relic account representative. 
-* To learn how you assign users permissions and access to specific accounts, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts). 
+A New Relic "account" can be considered a workspace. For example, you might have an account for a specific app, or a set of related hosts and services for a specific initiative or project, or you might have an account for a specific team.
 
-Having trouble accessing a specific account or feature? See [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data). 
+Our [Standard edition](https://newrelic.com/pricing) allows for a single account per organization. [Pro and Enterprise editions](https://newrelic.com/pricing) allow for multiple accounts per organizations. Currently you can't add accounts to your organization on your own. To add accounts, talk to your New Relic account representative. 
+
+### How users access accounts [#account-access]
+
+One effect of the organization structure is that you require a separate login for each New Relic organization on the new model. For example, if you're a contractor requiring access to several New Relic organizations that are on the new model, you'll need to sign out of one organization and into the other. If you have multiple logins associated with a single email address, you can [verify your account to see all associated logins](/docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems/#verify-email).
```

Comment:
Having tried this, I can verify it's a painful and tedious process every time :D would not recommend

Reply:
yes it is